content/zh-cn/docs/tutorials/kubernetes-basics/explore/explore-intro.html

```diff
@@ -274,13 +274,21 @@ <h3>在终端中显示应用</h3>
                 <p><code><b>export POD_NAME="$(kubectl get pods -o go-template --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}')"</b></code><br />
                 <code><b>echo Name of the Pod: $POD_NAME</b></code></p>
 
+                <p>在Windows系统中运行以下命令：</p>
+                <p><code><b>$POD_NAME="$(kubectl get pods -o go-template --template '{{range .items}}{{.metadata.name}}{{end}}')"</b></code><br />
+                <code><b>echo Name of the Pod: ${POD_NAME}</b></code></p>
```

Comment:
Please update the English upstream first, then we sync from the English version.

Reply:
Sure, it ready have a PR.

Reply:
can you link the en PR in here as well?

And once the english PR got merged, you might want to sync/rebase your PR

Reply:
Sure, link is here `https://github.com/kubernetes/website/pull/44016`